components/automate-ui/src/app/page-components/insight-attributes-dropdown/insight-attributes-dropdown.component.ts

```diff
@@ -1,87 +1,76 @@
-import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
```

Comment:
In order to allow state to be pushed into it from the reducer, I needed to refactor this component so that less state was being tracked/manipulated within it. Refactored bits are similar to projects-filter-dropdown.